docs/content/src/learn/tutorials/saas-integrations/amazons3/amazon-s3-bucket-service/README.md

```diff
@@ -0,0 +1,138 @@
+# Amazon S3 Bucket Service
+
+## About
+
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the 
+support of connectors. In this guide, we are mainly focusing on connecting to the Amazon Simple Storage Service API to create, store,download, and use data with other services.  
+
+The `wso2/amazons3` module allows you to perform the following operations.
```

Comment:
Seems like we are missing the operations.

Reply:
Fixed